L1Trigger/L1TGlobal/src/GlobalBoard.cc

```diff
@@ -971,10 +971,13 @@ void l1t::GlobalBoard::runFDL(edm::Event& iEvent,
   // prescale counters are reset at the beginning of the luminosity segment
   if (m_firstEv) {
     // prescale counters: numberPhysTriggers counters per bunch cross
-    m_prescaleCounterAlgoTrig.reserve(numberPhysTriggers * totalBxInEvent);
+    m_prescaleCounterAlgoTrig.reserve(totalBxInEvent);
```

Comment:
@cms-sw/l1-l2 please note this is the only line which should affect the default behavior (semiRandomInitialPSCounters = false)